include/swift/Option/Options.td

```diff
@@ -549,6 +549,11 @@ def disable_bridging_pch : Flag<["-"], "disable-bridging-pch">,
 def lto : Joined<["-"], "lto=">,
   Flags<[FrontendOption, NoInteractiveOption]>,
   HelpText<"Specify the LTO type to either 'llvm-thin' or 'llvm-full'">;
+  
+def lto_library : Separate<["-"], "lto-library">,
+  Flags<[FrontendOption, ArgumentIsPath, SwiftAPIExtractOption, SwiftSymbolGraphExtractOption,
+         SwiftAPIDigesterOption]>,
```

Comment:
```suggestion
  Flags<[FrontendOption, ArgumentIsPath, NoInteractiveOption]>,
```

Minor nit: we should restrict this to `swiftc` (NoInteractiveOption excludes `swift`)

Reply:
Makes sense to me